src/simmate/apps/clease/workflows/cluster_expansion.py

```diff
@@ -268,6 +270,17 @@ def run_config(
 
         # Once the loop exits above, we can write out ground state structures
         # https://clease.readthedocs.io/en/stable/aucu_probe_gs.html#generate-ground-state-structures
+        """
+        from ase.db import connect
```

Comment:
do you connect to the ASE database anywhere else? ASE uses SQLite so this might be the error you are seeing

Reply:
@scott-materials @laurenmm there's a good chance the "database is locked" error you're seeing is from ASE's database and not simmate's